ld_script_modern.ld

```diff
@@ -35,14 +35,13 @@ SECTIONS {
         src/m4a.o(.bss.code);
 
         /* COMMON starts at 0x30022A8 */
-        src/*.o(COMMON);
-        gflib/*.o(COMMON);
-        *libc.a:*.o(COMMON);
+        src/*.o(COMMON);            /**/
+        gflib/*.o(COMMON);          /**/
+        *libc.a:*.o(COMMON);        
```

Comment:
?

Reply:
Fighting syntax highlighting. Forgot to remove